AlfrescoApp/View Controllers/FileFolder View Controller/BaseLayoutAttributes.m

```diff
@@ -20,12 +20,51 @@
 
 @implementation BaseLayoutAttributes
 
+- (id)copyWithZone:(NSZone *)zone
+{
+    BaseLayoutAttributes *newAttributes = [super copyWithZone:zone];
+    if(newAttributes)
+    {
+        newAttributes.showDeleteButton = NO;
```

Comment:
A comment would be useful here explaining why this particular property is handled differently to all the others.